`brew install --HEAD lima` fails: `./_output/bin/limactl: No such file or directory `

```console
$ brew install --HEAD lima
==> Cloning https://github.com/lima-vm/lima.git
Updating /Users/suda/Library/Caches/Homebrew/lima--git
==> Checking out branch master
Already on 'master'
Your branch is up to date with 'origin/master'.
HEAD is now at c3e82cc Merge pull request #1164 from AkihiroSuda/dev2
==> make VERSION=HEAD-c3e82cc clean binaries
Last 15 lines from /Users/suda/Library/Logs/Homebrew/lima/01.make:
go: downloading github.com/kr/fs v0.1.0
go: downloading golang.org/x/crypto v0.1.0
go: downloading github.com/insomniacslk/dhcp v0.0.0-20220504074936-1ca156eafb9f
go: downloading github.com/apparentlymart/go-cidr v1.1.0
go: downloading github.com/google/gopacket v1.1.19
go: downloading golang.org/x/mod v0.6.0
go: downloading golang.org/x/time v0.0.0-20210723032227-1f47c861a9ac
go: downloading github.com/google/btree v1.0.1
go: downloading github.com/rivo/uniseg v0.2.0
go: downloading github.com/u-root/uio v0.0.0-20210528114334-82958018845c
go: downloading google.golang.org/grpc v1.47.0
go: downloading github.com/golang/protobuf v1.5.2
go: downloading google.golang.org/genproto v0.0.0-20220502173005-c8bf987b8c21
go: downloading google.golang.org/protobuf v1.28.0
go: downloading gopkg.in/djherbis/times.v1 v1.2.0

Do not report this issue to Homebrew/brew or Homebrew/core!

lima's formula was built from an unstable upstream --HEAD.
This build failure is expected behaviour.
Do not create issues about this on Homebrew's GitHub repositories.
Any opened issues will be immediately closed without response.
Do not ask for help from MacHomebrew on Twitter.
You may ask for help in Homebrew's discussions but are unlikely to receive a response.
Try to figure out the problem yourself and submit a fix as a pull request.
We will review it but may or may not accept it.

$ cat /Users/suda/Library/Logs/Homebrew/lima/01.make
2022-11-16 08:37:25 +0000                                                                                                           
                                                                  
make                                                              
VERSION=HEAD-c3e82cc                                              
clean                                                             
binaries                                                          
                                                                                                                                    
rm -rf _output vendor                                                                                                               
mkdir -p _output/bin                                              
# The hostagent must be compiled with CGO_ENABLED=1 so that net.LookupIP() in the DNS server                                        
codesign --entitlements vz.entitlements -s - ./_output/bin/limactl                                                                  
cp -a ./cmd/lima _output/bin/lima                                 
mkdir -p _output/bin                                              
# calls the native resolver library and not the simplistic version in the Go library.                                               
cp -a ./cmd/nerdctl.lima _output/bin/nerdctl.lima                 
./_output/bin/limactl: No such file or directory                  
cp -a cmd/apptainer.lima _output/bin/apptainer.lima                                                                                 
CGO_ENABLED=1 go build -ldflags="-s -w -X github.com/lima-vm/lima/pkg/version.Version=HEAD-c3e82cc" -o _output/bin/limactl ./cmd/limactl                                                                                                                                 
make: *** [codesign] Error 1                                      
make: *** Waiting for unfinished jobs....                         
go: downloading github.com/containerd/containerd v1.6.10                                                                            
go: downloading github.com/coreos/go-semver v0.3.0                
go: downloading github.com/AlecAivazis/survey/v2 v2.3.6
...
go: downloading google.golang.org/genproto v0.0.0-20220502173005-c8bf987b8c21                                                       
go: downloading google.golang.org/protobuf v1.28.0                                                                                                                                                                                                                       
go: downloading gopkg.in/djherbis/times.v1 v1.2.0                 
                                                                  
HOMEBREW_VERSION: 3.6.11-7-gaf1e1da                                                                                                 
ORIGIN: https://github.com/Homebrew/brew                          
HEAD: af1e1da9e0696a18e55c6f69c1adab9e23361423
...
```

https://github.com/Homebrew/homebrew-core/blob/8f62ec6c0659470fb63cc71c658bf378ad437ca4/Formula/lima.rb


macOS 13 Intel.

`brew install --build-from-source lima` works (Lima v0.13), so it looks like something was broken in the master branch recently.

Looks like Homebrew automatically injects `MAKEFLAGS=-j8`

```console
$ MAKEFLAGS=-j8 make
rm -rf _output vendor
mkdir -p _output/bin
# The hostagent must be compiled with CGO_ENABLED=1 so that net.LookupIP() in the DNS server
codesign --entitlements vz.entitlements -s - ./_output/bin/limactl
mkdir -p _output/bin
cp -a ./cmd/lima _output/bin/lima
cp -a ./cmd/nerdctl.lima _output/bin/nerdctl.lima
# calls the native resolver library and not the simplistic version in the Go library.
cp -a cmd/apptainer.lima _output/bin/apptainer.lima
cp -a cmd/docker.lima _output/bin/docker.lima
GOOS=linux GOARCH=amd64 CGO_ENABLED=0 go build -ldflags="-s -w -X github.com/lima-vm/lima/pkg/version.Version=v0.14.0-beta.0" -o _output/share/lima/lima-guestagent.Linux-x86_64 ./cmd/lima-guestagent
GOOS=linux GOARCH=arm64 CGO_ENABLED=0 go build -ldflags="-s -w -X github.com/lima-vm/lima/pkg/version.Version=v0.14.0-beta.0" -o _output/share/lima/lima-guestagent.Linux-aarch64 ./cmd/lima-guestagent
GOOS=linux GOARCH=riscv64 CGO_ENABLED=0 go build -ldflags="-s -w -X github.com/lima-vm/lima/pkg/version.Version=v0.14.0-beta.0" -o _output/share/lima/lima-guestagent.Linux-riscv64 ./cmd/lima-guestagent
cp -a cmd/podman.lima _output/bin/podman.lima
CGO_ENABLED=1 go build -ldflags="-s -w -X github.com/lima-vm/lima/pkg/version.Version=v0.14.0-beta.0" -o _output/bin/limactl ./cmd/limactl
./_output/bin/limactl: No such file or directory
make: *** [codesign] Error 1
make: *** Waiting for unfinished jobs....
chmod 644 _output/share/lima/lima-guestagent.Linux-aarch64
chmod 644 _output/share/lima/lima-guestagent.Linux-x86_64
chmod 644 _output/share/lima/lima-guestagent.Linux-riscv64
```

--- a/Makefile
+++ b/Makefile
@@ -20,6 +20,8 @@ VERSION_TRIMMED := $(VERSION:v%=%)
 
 GO_BUILD := $(GO) build -ldflags="-s -w -X $(PACKAGE)/pkg/version.Version=$(VERSION)"
 
+.NOTPARALLEL:
+
 .PHONY: all
 all: binaries
 

